spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/config/KafkaBinderDefaultProperties.java

```diff
@@ -25,18 +25,12 @@
 @ConfigurationProperties(value = "spring.cloud.stream.binder.kafka.default")
 public class KafkaBinderDefaultProperties {
```

Comment:
Unless there is a specific reason, I think it would be nice to avoid `Default` and having the name [Kafka/Rabbit etc.,]`BinderProperties`  based on the discussion from [here] (https://github.com/spring-cloud/spring-cloud-stream/pull/409/files#r55822262)